package.json

```diff
@@ -27,26 +23,22 @@
     "url": "https://github.com/feathersjs/feathers-authentication/issues"
   },
   "engines": {
-    "node": ">= 4"
+    "node": ">= 6"
   },
   "scripts": {
     "start": "node example/app",
-    "prepublish": "npm run compile",
     "publish": "git push origin && git push origin --tags",
     "release:patch": "npm version patch && npm publish",
     "release:minor": "npm version minor && npm publish",
     "release:major": "npm version major && npm publish",
-    "release:pre": "npm publish --tag next",
+    "release:pre": "npm publish --tag pre",
     "changelog": "github_changelog_generator && git add CHANGELOG.md && git commit -am \"Updating changelog\"",
-    "compile": "rimraf lib/ && babel -d lib/ src/",
-    "watch": "babel --watch -d lib/ src/",
     "lint": "semistandard --fix",
     "mocha": "mocha --opts mocha.opts",
-    "test": "npm run compile && npm run lint && npm run coverage && nsp check",
+    "test": "npm run lint && npm run coverage",
```

Comment:
Are we dropping the `nsp` check? Does Greenkeeper cover us there?

Reply:
Nah, good call, added it back.